docs/source/upgrade/example/coin-upgrade-trigger/daml/UpgradeTrigger.daml

```diff
@@ -40,16 +42,16 @@ triggerRule issuer acs _ _ _ = do
         [toAnyContractId coinCid]
 -- TRIGGER_RULE_END
 
--- TODO (MK) The Bazel rule atm doesn’t run this scenario, we should fix that.
-test = scenario do
-  alice <- getParty "Alice"
-  bob <- getParty "Bob"
+-- TODO (MK) The Bazel rule atm doesn’t run this script, we should fix that.
+test = script do
+  alice <- Script.allocateParty "Alice"
+  bob <- Script.allocateParty "Bob"
 
-  coinProposal <- submit alice $ create (CoinProposal alice bob)
-  coin <- submit bob $ exercise coinProposal CoinProposal_Accept
+  coinProposal <- submit alice $ Script.createCmd (CoinProposal alice bob)
```

Comment:
Side note: It is starting to become attractive to overload `createCmd` and friends so we don’t get collisions between daml triggers and daml script. I expect in almost all cases type inference will work fine here but needs someone to actually try it out.

Reply:
I would expect the same, particularly when you use a helper function like the newly introduces `script`.